### ISSUE TITLE
Incompatible with Sprockets-Rails 3.x

### ISSUE THREAD

**Issue by lowjoel** [ai/jquery-cdn#2]
`config.assets.compile` is now nil by default in 3.x. See their README.

This prevents apps using jquery-cdn from booting:
```
NoMethodError: undefined method `prepend_path' for nil:NilClass
/home/travis/build/Coursemology/coursemology2/vendor/bundle/ruby/2.2.0/gems/jquery-cdn-2.1.4/lib/jquery-cdn/railties.rb:20:in `block in <class:Railtie>'
/home/travis/build/Coursemology/coursemology2/vendor/bundle/ruby/2.2.0/gems/railties-4.2.5/lib/rails/initializable.rb:30:in `instance_exec'
/home/travis/build/Coursemology/coursemology2/vendor/bundle/ruby/2.2.0/gems/railties-4.2.5/lib/rails/initializable.rb:30:in `run'
/home/travis/build/Coursemology/coursemology2/vendor/bundle/ruby/2.2.0/gems/railties-4.2.5/lib/rails/initializable.rb:55:in `block in run_initializers'
/home/travis/build/Coursemology/coursemology2/vendor/bundle/ruby/2.2.0/gems/railties-4.2.5/lib/rails/initializable.rb:54:in `run_initializers'
/home/travis/build/Coursemology/coursemology2/vendor/bundle/ruby/2.2.0/gems/railties-4.2.5/lib/rails/application.rb:352:in `initialize!'
/home/travis/build/Coursemology/coursemology2/config/environment.rb:5:in `<top (required)>'
```

**Comment by ai** [comment]
I accepted your PR and test it in Rails 4 with Sprockets 2. Can you test it in your Rails 5 project?

Just replace `gem 'jquery-cdn'` to `gem 'jquery-cdn', github: 'ai/jquery-cdn'`.

**Comment by lowjoel** [comment]
I'm actually using Rails 4. They just released sprockets-rails 3 (presumably for Rails 5) but should work with Rails 4. I just did a `bundle update`.

**Comment by lowjoel** [comment]
Seems to work, I think.

**Comment by ai** [comment]
Thanks! Released in 2.1.4.1.
